### DISCUSSION TITLE
Adguard stats into influxdb

### DISCUSSION THREAD

**Discussion by icornish72** [AdguardTeam/AdGuardHome#4691] [Q&A]
Hi,
I've been looking at how we can get stats via the JSON API into Influxdb. I've tried to search, and haven't found much already out there.
Is there any existing work out there or am I starting from scratch?

**Comment by emlimap** [comment]
abstractvector posted a telegraf rule to process & ingest query log files into influxdb in this issue which might of use to you https://github.com/AdguardTeam/AdGuardHome/issues/573#issuecomment-760167925
